packages/lighthouse-spa/src/app/home/home.component.ts

```diff
@@ -0,0 +1,176 @@
+import { Component, OnInit } from '@angular/core';
+import { environment } from '../../environments/environment';
+import { AppService } from '../app.service';
+const Ansi = require( 'ansi-to-html' );
+@Component( {
+  selector: 'app-home',
+  templateUrl: './home.component.html',
+  styleUrls: [ './home.component.scss' ]
+} )
+export class HomeComponent implements OnInit {
+  document: Document | any;
```

Comment:
Do not use `Type | any` it defeats the purpose of type, and works the same as `any`. Always try to use strict types. If it shows any errors elsewhere, you should fix it in the appropriate place instead of changing the type to `any`

Same for the other properties.

```suggestion
  document: Document;
```

Reply:
Updated the types